draft-ietf-quic-transport.md

```diff
@@ -621,10 +621,19 @@ packet. It also sets the Token field to the token provided in the Retry. The
 client MUST NOT change the Source Connection ID because the server could include
 the connection ID as part of its token validation logic (see {{tokens}}).
 
+The Initial packet sent by the client in response to a Retry packet is subject
+to the same restrictions as the first Initial packet, with the exception of the
+value of the Destination Connection ID and Token fields, which are set as
+described here.  A client can either reuse the cryptographic handshake message
+or construct a new one at its discretion.  Any subsequent Initial packets from
+the client MUST use the same connection ID values, and MUST NOT include the
```

Comment:
I thought the Retry now must change the CID?

Reply:
Yeah, I meant "same as the one from the Retry"  clarified.